react/react-training/src/redux/login/actions.js

```diff
@@ -0,0 +1,38 @@
+import UserService from '@services/UserService.js';
+import { MSG_UNKNOWN_ID, ERROR_READING_RESPONSE, ERROR_LOGIN_SERVICE } from '@screens/Login/validation';
+
+export const actionTypes = {
+  login: 'LOGIN',
+  loginSuccess: 'LOGIN_SUCCESS',
+  loginFailure: 'LOGIN_FAILURE'
+};
+
+export const actionsCreators = {
+  login: (username, password) => async dispatch => {
+    dispatch({ type: actionTypes.login });
+    const response = await UserService.login(username, password);
+    if (response && response.data) {
```

Comment:
with asking for `(response.ok)` is enough. I'd delete this if

Reply:
Hola @WFolini, te pregunto…
Me dices que puedo eliminar la linea `if (response && response.data) {`
Sin embargo es necesaria.. Si el servicio está fuera de línea no hay `response` y por tanto la app saca este error:
<img width="555" alt="gabo s tic tac toe 2018-11-15 16-08-28" src="https://user-images.githubusercontent.com/656353/48582038-75727d80-e8f1-11e8-99df-b0045e38b721.png">